test/unit/attributes.js

```diff
@@ -1328,6 +1328,34 @@ test( "toggleClass(Function[, boolean]) with incoming value", function() {
 	ok( !e.is(".test"), "Assert class not present" );
 });
 
+test( "toggleClass(String[, Function])", function() {
+	expect( 9 );
+
+	var el = jQuery("<div><span data-tab='1'>One</span><span data-tab='2'>Two</span><span data-tab='3'>Three</span></div>"),
+		elChildren = el.children(),
+		currentTab = 2,
+		testFn = function( i, e ) {
+			return jQuery(e).data("tab") !== currentTab;
```

Comment:
This test is dependent on the `data` module now. I think it should be independent; please switch this to a regular `getAttribute('data-tab')`.